streaming/src/main/scala/org/apache/carbondata/streaming/parser/FieldConverter.scala

```diff
@@ -24,6 +24,7 @@ import java.util
 import org.apache.carbondata.core.constants.CarbonCommonConstants
 
 object FieldConverter {
+  val exceedErrorMsg = "Dataload failed, String length cannot exceed "
```

Comment:
Better to rename `exceedErrorMsg` to `stringLengthExceedErrorMsg`

Reply:
done